devices/philips.js

```diff
@@ -3,987 +3,588 @@ const fz = {...require('../converters/fromZigbee'), legacy: require('../lib/lega
 const tz = require('../converters/toZigbee');
 const ota = require('../lib/ota');
 const reporting = require('../lib/reporting');
-const globalStore = require('../lib/store');
 const philips = require('../lib/philips');
-const utils = require('../lib/utils');
-const libColor = require('../lib/color');
-const herdsman = require('zigbee-herdsman');
 const e = exposes.presets;
 const ea = exposes.access;
 
-// Make sure extend.light_* is not used (hueExtend should be used instead)
+// Make sure extend.light_* is not used (philips.hueExtend should be used instead)
 const extendDontUse = require('../lib/extend');
 const extend = {switch: extendDontUse.switch};
```

Comment:
`const extendDontUse = require('../lib/extend');` can be removed, when changing `onst extend = {switch: extendDontUse.switch};` to `onst extend = {switch: require('../lib/extend').switch};`